mcs/tools/mdoc/Test/html.expected/Mono.DocTest.Generic/Extensions.html

```diff
@@ -285,14 +285,14 @@ <h2 class="Section">Public Methods</h2>
     <div class="Members" id="T:Mono.DocTest.Generic.Extensions:Members">
       <h2 class="Section" id="MemberDetails">Member Details</h2>
       <div class="SectionBox" id="_MemberDetails">
-        <h3 id="M:Mono.DocTest.Generic.Extensions.Bar``1(Mono.DocTest.Generic.IFoo{``0},System.String)">Bar&lt;T&gt; Generic Method</h3>
```

Comment:
The problem with this is that it *breaks all existing anchor references*.

*Additionally*, it makes the type parameter name part of the "documentation ABI" -- that is, changing type parameter names will break the documentation, as existing member references will have different names. This is not currently a C# ABI break (or any other language that I know of).